R/webshot.R

```diff
@@ -308,7 +329,7 @@ new_session_screenshot <- function(
     })$
     finally(function() {
       # Close down the session if we successfully started one
-      if (!is.null(s)) s$close()
+      # if (!is.null(s)) s$close()
```

Comment:
Just wondering, why is this commented out?

Reply:
Thanks for spotting this! I need to restore the line. During debug I took this line out so I could access the screenshot session to see what was going on